code-challenges.test.js

```diff
@@ -14,12 +14,89 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
+// Pseudocode:
+    // Set describe, it, and expect statements. 
+    // create a function that takes in >2 and returns fibonacci array
+    // use .toEqual to make the test pass
+    // input: array
+    // Expected output 1: [1, 1, 2, 3, 5, 8]
+    // Expected output 2: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+    // create a function () => {} const
+    // taking a parameter that represents array seqence
+    // use .length and .push method on that array parameter 
+    // evaluate the length of each element of the array parameter
+    // calculate if the length of each element of array parameter  by adding the previous 2 elements and coming up with the sum in the next
+    // create a conditional to evaluate the fibonacci sequence
+    // return an array provided by the .push that contains the next number in which consequential numbers are the sum of the previous 2 numbers.
```

Comment:
You don't need to pseudo code your test.
Here are some notes about this pseudo code:
- Line 25: The parameter represents a number
- Line 26: The code doesn't use .length or .push on the parameter, it's using it on the created variable holding an array
- Line 28: this line doesn't make sense.
- Line 29: The conditional doesn't evaluate the fib sequence, it evaluates whether the argument is 0, 1, or 2 and if not, then runs logic
- Line 30: The .push method does not provide an array

Overall the pseudo code does not accurately explain what's going on in the code, and shows that you don't understand the answer you found.